src/query/frontend/ast/cypher_main_visitor.cpp

```diff
@@ -1334,6 +1340,30 @@ antlrcpp::Any CypherMainVisitor::visitRevokePrivilege(MemgraphCypher::RevokePriv
   return auth;
 }
 
+/**
+ * @return AuthQuery*
+ */
+antlrcpp::Any CypherMainVisitor::visitLabelPrivilegeList(MemgraphCypher::LabelPrivilegeListContext *ctx) {
+  std::unordered_map<AuthQuery::LabelPrivilege, std::vector<std::string>> result;
+  for (auto *privilege : ctx->singlePrivilege()) {
```

Comment:
Maybe it would be a good naming "singleLabelPrivilege", in all the places

Reply:
We can iterate over naming, i agree